src/JoyOI.Monitor/Controllers/Blog/NewArticleController.cs

```diff
@@ -24,7 +24,7 @@ public class NewArticleController : ChartController
             }
             var scaling = new ChartScaling(start, end, interval);
             return Json(await GetChartData(
-                FORUM,
+                BLOG,
```

Comment:
Please convert all Full-upper-cased constants into First-letter-upper-case. Full-upper-case is not a `c#` coding practice.